### ISSUE TITLE
RareIce "Could not initialize class" error fills logs 

### ISSUE THREAD

**Issue by rirze** [xJon/Hexxit-II#12]
**Basic details**
- **Are you using Java 8, 64-bit?**: **no** / yes
    - I'm using Java 8, for ARM, running on 3 GB RAM (OpenJDK Runtime Environment Corretto-8.302.08.1 (build 1.8.0_302-b08))
    	- I totally understand if this is not supported. 
- **Did you add any additional mods**: yes (detail) / **no**
- **Crash/latest log link (if applicable)**: link to log / not applicable / **shown below**
<details>
<summary> Relevant part of logs</summary>
<br>

```
[17:58:47] [Server thread/FATAL] [FML]: ---- Minecraft Crash Report ----
// Why did you do that?

Time: 2021-10-24 17:58:47 UTC
Description: Ticking block entity

java.lang.NoClassDefFoundError: Could not initialize class me.shedaniel.rareice.forge.loot.LootGenerator
    at me.shedaniel.rareice.forge.blocks.entities.RareIceTileEntity.update(RareIceTileEntity.java:116)
    at net.minecraft.world.World.redirect$zzk000$tileEntityUpdate(World.java:4137)
    at net.minecraft.world.World.updateEntities(World.java:1838)
    at net.minecraft.world.WorldServer.updateEntities(WorldServer.java:613)
    at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:767)
    at net.minecraft.server.dedicated.DedicatedServer.updateTimeLightAndEntities(DedicatedServer.java:397)
    at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
    at java.lang.Thread.run(Thread.java:748)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Block entity being ticked --
  Name: rare-ice:rare_ice // me.shedaniel.rareice.forge.blocks.entities.RareIceTileEntity
  Block type: ID #1974 (tile.rare-ice:rare_ice // me.shedaniel.rareice.forge.blocks.RareIceBlock // rare-ice:rare_ice)
  Block data value: 0 / 0x0 / 0b0000
  Block location: World: (1,62,315), Chunk: (at 1,3,11 in 0,19; contains blocks 0,0,304 to 15,255,319), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
  Actual block type: ID #1974 (tile.rare-ice:rare_ice // me.shedaniel.rareice.forge.blocks.RareIceBlock // rare-ice:rare_ice)
  Actual block data value: 0 / 0x0 / 0b0000
  Block Entity NBT: {x:1,y:62,z:315,Items:[],ItemLocations:[],id:"rare-ice:rare_ice",RevertDelay:0}

-- System Details --
  Minecraft Version: 1.12.2
  Operating System: Linux (aarch64) version 4.14.238-182.422.amzn2.aarch64
  Java Version: 1.8.0_302, Amazon.com Inc.
  Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Amazon.com Inc.
  Memory: 455804288 bytes (434 MB) / 1438420992 bytes (1371 MB) up to 3113877504 bytes (2969 MB)
  JVM Flags: 1 total; -Xmx3G
  IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
  FML: MCP 9.42 Powered by Forge 14.23.5.2855 179 mods loaded, 179 mods active
       States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
       
       | State  | ID                                   | Version                  | Source                                         | Signature                                |
       |:------ |:------------------------------------ |:------------------------ |:---------------------------------------------- |:---------------------------------------- |
       | LCHIJA | minecraft                            | 1.12.2                   | minecraft.jar                                  | None                                     |
       | LCHIJA | mcp                                  | 9.42                     | minecraft.jar                                  | None                                     |
       | LCHIJA | FML                                  | 8.0.99.99                | forge-1.12.2-14.23.5.2855.jar                  | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | LCHIJA | forge                                | 14.23.5.2855             | forge-1.12.2-14.23.5.2855.jar                  | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | LCHIJA | ivtoolkit                            | 1.3.3-1.12               | minecraft.jar                                  | None                                     |
       | LCHIJA | OpenEye                              | 0.8                      | OpenEye-1.12.2-0.8.jar                         | None                                     |
       | LCHIJA | openmodscore                         | 0.12.2                   | minecraft.jar                                  | None                                     |
       | LCHIJA | foamfixcore                          | 7.7.4                    | minecraft.jar                                  | None                                     |
       | LCHIJA | obfuscate                            | 0.4.2                    | minecraft.jar                                  | None                                     |
       | LCHIJA | srm-hooks                            | 1.12.2-1.0.0             | minecraft.jar                                  | None                                     |
       | LCHIJA | randompatches                        | 1.12.2-1.22.1.10         | randompatches-1.12.2-1.22.1.10.jar             | None                                     |
       | LCHIJA | bspkrscore                           | 7.6.0.1                  | [1.12]bspkrsCore-universal-7.6.0.1.jar         | None                                     |
       | LCHIJA | treecapitator                        | 1.43.0                   | [1.12]TreeCapitator-client-1.43.0.jar          | None                                     |
       | LCHIJA | additionalbanners                    | 1.1.77                   | AdditionalBanners-1.12.2-1.1.77.jar            | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHIJA | akashictome                          | 1.2-12                   | AkashicTome-1.2-12.jar                         | None                                     |
       | LCHIJA | crafttweaker                         | 4.1.20                   | CraftTweaker2-1.12-4.1.20.662.jar              | None                                     |
       | LCHIJA | jei                                  | 4.16.1.302               | jei_1.12.2-4.16.1.302.jar                      | None                                     |
       | LCHIJA | appleskin                            | 1.0.14                   | AppleSkin-mc1.12-1.0.14.jar                    | None                                     |
       | LCHIJA | asmc                                 | 1.0.2                    | ASMC-1.12.2-1.0.2.jar                          | None                                     |
       | LCHIJA | attributefix                         | 1.0.4                    | AttributeFix-1.12.2-1.0.4.jar                  | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHIJA | quark                                | r1.6-179                 | Quark-r1.6-179.jar                             | None                                     |
       | LCHIJA | autoreglib                           | 1.3-32                   | AutoRegLib-1.3-32.jar                          | None                                     |
       | LCHIJA | backpack                             | 3.0.2                    | backpack-3.0.2-1.12.2.jar                      | None                                     |
       | LCHIJA | backups                              | 1.5.7                    | Backups-1.12.2-1.5.7.jar                       | None                                     |
       | LCHIJA | badwithernocookiereloaded            | 1.12.2-3.4.18            | badwithernocookiereloaded-1.12.2-3.4.18.jar    | None                                     |
       | LCHIJA | battletowers                         | 1.6.5                    | BattleTowers-1.12.2.jar                        | None                                     |
       | LCHIJA | baubles                              | 1.5.2                    | Baubles-1.12-1.5.2.jar                         | None                                     |
       | LCHIJA | betterbuilderswands                  | 0.13.2                   | BetterBuildersWands-1.12.2-0.13.2.271.jar      | None                                     |
       | LCHIJA | betterinvisibility                   | 1.0.1                    | betterinvisibility-1.0.1.jar                   | None                                     |
       | LCHIJA | bettermineshafts                     | 1.12.2-2.2.1             | BetterMineshaftsForge-1.12.2-2.2.1.jar         | None                                     |
       | LCHIJA | bibliocraft                          | 2.4.5                    | BiblioCraft[v2.4.5][MC1.12.2].jar              | None                                     |
       | LCHIJA | birdsnests                           | 2.1.0                    | BirdsNests1.12.2-2.1.0.jar                     | None                                     |
       | LCHIJA | blockcraftery                        | 1.12.2-1.3.1             | blockcraftery-1.12.2-1.3.1.jar                 | None                                     |
       | LCHIJA | bookshelf                            | 2.3.590                  | Bookshelf-1.12.2-2.3.590.jar                   | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHIJA | forgelin                             | 1.8.4                    | Forgelin-1.8.4.jar                             | None                                     |
       | LCHIJA | bountiful                            | 2.2.2                    | Bountiful-2.2.2.jar                            | None                                     |
       | LCHIJA | cakechomps                           | 1.12.2-1.0               | cakechomps-1.12.2-1.0.jar                      | 5d5b8aee896a4f5ea3f3114784742662a67ad32f |
       | LCHIJA | chameleon                            | 1.12-4.1.3               | Chameleon-1.12-4.1.3.jar                       | None                                     |
       | LCHIJA | chesttransporter                     | 2.8.8                    | ChestTransporter-1.12.2-2.8.8.jar              | None                                     |
       | LCHIJA | chisel                               | MC1.12.2-1.0.2.45        | Chisel-MC1.12.2-1.0.2.45.jar                   | None                                     |
       | LCHIJA | chiselsandbits                       | 14.33                    | chiselsandbits-14.33.jar                       | None                                     |
       | LCHIJA | chococraftplus                       | 4.5.6                    | ChocoCraftPlus-1.12.2-4.5.6.jar                | None                                     |
       | LCHIJA | geckolib3                            | 3.0.0                    | geckolib-forge-1.12.2-3.0.0-beta.8.jar         | None                                     |
       | LCHIJA | phosphor-lighting                    | 1.12.2-0.2.7             | phosphor-forge-mc1.12.2-0.2.7-universal.jar    | None                                     |
       | LCHIJA | cqrepoured                           | 2.5.3B                   | Chocolate_Quest_Repoured-1.12.2-2.5.3B.jar     | None                                     |
       | LCHIJA | chunkgenlimit                        | 1.1                      | chunkgenlimiter-1.1.jar                        | None                                     |
       | LCHIJA | classiccombat                        | 1.0.1                    | ClassicCombat-1.0.1.jar                        | None                                     |
       | LCHIJA | codechickenlib                       | 3.2.3.358                | CodeChickenLib-1.12.2-3.2.3.358-universal.jar  | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
       | LCHIJA | redstoneflux                         | 2.1.1                    | RedstoneFlux-1.12-2.1.1.1-universal.jar        | None                                     |
       | LCHIJA | cofhcore                             | 4.6.6                    | CoFHCore-1.12.2-4.6.6.1-universal.jar          | None                                     |
       | LCHIJA | cofhworld                            | 1.4.0                    | CoFHWorld-1.12.2-1.4.0.1-universal.jar         | None                                     |
       | LCHIJA | k4lib                                | 1.12.1-2.1.81            | k4lib-1.12.1-2.1.81-universal.jar              | None                                     |
       | LCHIJA | colorchat                            | 1.12.1-2.0.43            | colorchat-1.12.1-2.0.43-universal.jar          | None                                     |
       | LCHIJA | colytra                              | 1.2.0.4                  | colytra-1.12.2-1.2.0.4.jar                     | 2484ef4d131fdc0dca0647aa21b7b944ddb935a1 |
       | LCHIJA | combustfish                          | 1.0.3.4                  | combustfish-1.12.2-1.0.3.4.jar                 | 2484ef4d131fdc0dca0647aa21b7b944ddb935a1 |
       | LCHIJA | morpheus                             | 1.12.2-3.5.106           | Morpheus-1.12.2-3.5.106.jar                    | None                                     |
       | LCHIJA | comforts                             | 1.4.1.3                  | comforts-1.12.2-1.4.1.3.jar                    | 2484ef4d131fdc0dca0647aa21b7b944ddb935a1 |
       | LCHIJA | mantle                               | 1.12-1.3.3.55            | Mantle-1.12-1.3.3.55.jar                       | None                                     |
       | LCHIJA | twilightforest                       | 3.11.1021                | twilightforest-1.12.2-3.11.1021-universal.jar  | None                                     |
       | LCHIJA | tconstruct                           | 1.12.2-2.13.0.183        | TConstruct-1.12.2-2.13.0.183.jar               | None                                     |
       | LCHIJA | conarm                               | 1.2.5.10                 | conarm-1.12.2-1.2.5.10.jar                     | b33d2c8df492beff56d1bbbc92da49b8ab7345a1 |
       | LCHIJA | consolefilter                        | 1.1.1                    | ConsoleFilter-1.12.2-1.1.1.jar                 | None                                     |
       | LCHIJA | cosmeticarmorreworked                | 1.12.2-v5a               | CosmeticArmorReworked-1.12.2-v5a.jar           | aaaf83332a11df02406e9f266b1b65c1306f0f76 |
       | LCHIJA | crafttweakerjei                      | 2.0.3                    | CraftTweaker2-1.12-4.1.20.662.jar              | None                                     |
       | LCHIJA | creeperconfetti                      | 1.4.2                    | creeperconfetti-1.4.2.jar                      | None                                     |
       | LCHIJA | culinaryconstruct                    | 1.3.4                    | culinaryconstruct-1.3.4.jar                    | 2484ef4d131fdc0dca0647aa21b7b944ddb935a1 |
       | LCHIJA | customspawner                        | 3.11.4                   | CustomMobSpawner-3.11.5.jar                    | None                                     |
       | LCHIJA | customstartinggear                   | 1.2.0                    | CustomStartingGear-1.12-1.2.0.9-universal.jar  | None                                     |
       | LCHIJA | damagetilt                           | 0.1.0                    | DamageTilt-1.12.2-0.1.0.jar                    | None                                     |
       | LCHIJA | deadlyworld                          | 1.1.1_for_mc1.12.2       | deadlyworld-1.1.1_for_mc1.12.2.jar             | None                                     |
       | LCHIJA | deathcounter                         | 1.1.0                    | DeathCounter-1.12.2-1.1.0.jar                  | None                                     |
       | LCHIJA | debark                               | 0.1.2                    | Debark-0.1.2.jar                               | None                                     |
       | LCHIJA | dimdoors                             | 3.0.9+287                | DimensionalDoors-3.0.9-287.jar                 | None                                     |
       | LCHIJA | mocreatures                          | 12.0.5                   | DrZharks MoCreatures Mod-12.0.5.jar            | None                                     |
       | LCHIJA | dupefixproject                       | 3.1.6                    | DupeFixProject-1.12.2-3.1.6.jar                | None                                     |
       | LCHIJA | element                              | 1.0.5                    | Elementary Staffs 1.0.6.jar                    | None                                     |
       | LCHIJA | elenaidodge                          | 1.13.2                   | elenaidodge-1.12.2-1.13.2.jar                  | None                                     |
       | LCHIJA | eplus                                | 5.0.176                  | EnchantingPlus-1.12.2-5.0.176.jar              | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHIJA | gunpowderlib                         | 1.12.2-1.1               | GunpowderLib-1.12.2-1.1.jar                    | 4ffa87db52cf086d00ecc4853a929367b1c39b5c |
       | LCHIJA | thermalfoundation                    | 2.6.7                    | ThermalFoundation-1.12.2-2.6.7.1-universal.jar | None                                     |
       | LCHIJA | exchangers                           | 1.12.2-2.10.1            | Exchangers-1.12.2-2.10.1.jar                   | 4ffa87db52cf086d00ecc4853a929367b1c39b5c |
       | LCHIJA | eyesinthedarkness                    | 0.1.0                    | EyesInTheDarkness-0.1.0.jar                    | None                                     |
       | LCHIJA | fairylights                          | 2.1.10                   | fairylights-2.2.0-1.12.2.jar                   | None                                     |
       | LCHIJA | fantasticlib                         | 1.12.2.047               | FantasticLib-1.12.2.047.jar                    | None                                     |
       | LCHIJA | farmingforblockheads                 | 3.1.28                   | FarmingForBlockheads_1.12.2-3.1.28.jar         | None                                     |
       | LCHIJA | farseek                              | 2.5.1                    | Farseek-1.12-2.5.1.jar                         | None                                     |
       | LCHIJA | fastleafdecay                        | v14                      | FastLeafDecay-v14.jar                          | None                                     |
       | LCHIJA | fastbench                            | 1.7.3                    | FastWorkbench-1.12.2-1.7.3.jar                 | None                                     |
       | LCHIJA | fencejumper                          | 1.0.5                    | fencejumper-1.12-1.0.5.jar                     | None                                     |
       | LCHIJA | foamfix                              | 0.10.14-1.12.2           | foamfix-0.10.14-1.12.2.jar                     | None                                     |
       | LCHIJA | forgemultipartcbe                    | 2.6.2.83                 | ForgeMultipart-1.12.2-2.6.2.83-universal.jar   | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
       | LCHIJA | microblockcbe                        | 2.6.2.83                 | ForgeMultipart-1.12.2-2.6.2.83-universal.jar   | None                                     |
       | LCHIJA | minecraftmultipartcbe                | 2.6.2.83                 | ForgeMultipart-1.12.2-2.6.2.83-universal.jar   | None                                     |
       | LCHIJA | growthcraft                          | 4.2.2                    | growthcraft-1.12.2-4.2.2.1.jar                 | None                                     |
       | LCHIJA | growthcraft_fishtrap                 | 4.2.2                    | growthcraft-1.12.2-4.2.2.1.jar                 | None                                     |
       | LCHIJA | growthcraft_apples                   | 4.2.2                    | growthcraft-1.12.2-4.2.2.1.jar                 | None                                     |
       | LCHIJA | growthcraft_cellar                   | 4.2.2                    | growthcraft-1.12.2-4.2.2.1.jar                 | None                                     |
       | LCHIJA | growthcraft_grapes                   | 4.2.2                    | growthcraft-1.12.2-4.2.2.1.jar                 | None                                     |
       | LCHIJA | growthcraft_bamboo                   | 4.2.2                    | growthcraft-1.12.2-4.2.2.1.jar                 | None                                     |
       | LCHIJA | growthcraft_bees                     | 4.2.2                    | growthcraft-1.12.2-4.2.2.1.jar                 | None                                     |
       | LCHIJA | growthcraft_rice                     | 4.2.2                    | growthcraft-1.12.2-4.2.2.1.jar                 | None                                     |
       | LCHIJA | growthcraft_hops                     | 4.2.2                    | growthcraft-1.12.2-4.2.2.1.jar                 | None                                     |
       | LCHIJA | growthcraft_milk                     | 4.2.2                    | growthcraft-1.12.2-4.2.2.1.jar                 | None                                     |
       | LCHIJA | helpfixer                            | 1.12.1-1.5.18            | HelpFixer-1.12.1-1.5.18.jar                    | None                                     |
       | LCHIJA | placebo                              | 1.6.0                    | Placebo-1.12.2-1.6.0.jar                       | None                                     |
       | LCHIJA | hexxitgear                           | 2.8.0                    | Hexxit Gear-1.12.2-2.8.0.jar                   | None                                     |
       | LCHIJA | hexxitworld                          | 1.0.0                    | HexxitWorld-1.12.2-1.0.0.jar                   | None                                     |
       | LCHIJA | waila                                | 1.8.26                   | Hwyla-1.8.26-B41_1.12.2.jar                    | None                                     |
       | LCHIJA | ichunutil                            | 7.2.2                    | iChunUtil-1.12.2-7.2.2.jar                     | 4db5c2bd1b556f252a5b8b54b256d381b2a0a6b8 |
       | LCHIJA | ido                                  | 2.0.0-B                  | ido-1.12.2-2.0.0-B.jar                         | None                                     |
       | LCHIJA | infernalmobs                         | 1.7.6                    | InfernalMobs-1.12.2.jar                        | None                                     |
       | LCHIJA | ironchest                            | 1.12.2-7.0.67.844        | ironchest-1.12.2-7.0.72.847.jar                | None                                     |
       | LCHIJA | itemdamagerebalancer                 | 1.12.2.003               | ItemDamageRebalancer-1.12.2.003.jar            | None                                     |
       | LCHIJA | itlt                                 | 1.0.4                    | itlt-1.12.x-1.0.4.jar                          | None                                     |
       | LCHIJA | jepb                                 | 1.2.1                    | jepb-1.12-1.2.1.jar                            | None                                     |
       | LCHIJA | jexclusives                          | 1.1.0                    | JonsExclusives-MC1.12.2-v1.1.0.jar             | None                                     |
       | LCHIJA | jrftl                                | 1.1                      | JRFTL[1.12.2]-1.1.jar                          | None                                     |
       | LCHIJA | kleeslabs                            | 5.4.12                   | KleeSlabs_1.12.2-5.4.12.jar                    | None                                     |
       | LCHIJA | loottweaker                          | 0.3.0                    | LootTweaker-0.3.0+MC1.12.2.jar                 | None                                     |
       | LCHIJA | lostbooks                            | 2.0.3                    | LostBooks-II-1.12.2-2.0.3.jar                  | None                                     |
       | LCHIJA | lunatriuscore                        | 1.2.0.42                 | LunatriusCore-1.12.2-1.2.0.42-universal.jar    | None                                     |
       | LCHIJA | mcmultipart                          | 2.5.3                    | MCMultiPart-2.5.3.jar                          | None                                     |
       | LCHIJA | testdummy                            | 1.12                     | MmmMmmMmmMmm-1.12-1.14.jar                     | None                                     |
       | LCHIJA | natura                               | 1.12.2-4.3.2.69          | natura-1.12.2-4.3.2.69.jar                     | None                                     |
       | LCHIJA | naturescompass                       | 1.8.5                    | NaturesCompass-1.12.2-1.8.5.jar                | None                                     |
       | LCHIJA | plustic                              | 8.0.1.0                  | plustic-8.0.1.0.jar                            | None                                     |
       | LCHIJA | netherendingores                     | 1.12.2-1.4.2             | Netherending-Ores-1.12.2-1.4.2.jar             | None                                     |
       | LCHIJA | netherportalfix                      | 5.3.17                   | NetherPortalFix_1.12.1-5.3.17.jar              | None                                     |
       | LCHIJA | recipehandler                        | 0.13                     | NoMoreRecipeConflict-0.13(1.12.2).jar          | None                                     |
       | LCHIJA | nyx                                  | 1.4.2                    | Nyx-1.4.2.jar                                  | None                                     |
       | LCHIJA | openmods                             | 0.12.2                   | OpenModsLib-1.12.2-0.12.2.jar                  | d2a9a8e8440196e26a268d1f3ddc01b2e9c572a5 |
       | LCHIJA | openblocks                           | 1.8.1                    | OpenBlocks-1.12.2-1.8.1.jar                    | d2a9a8e8440196e26a268d1f3ddc01b2e9c572a5 |
       | LCHIJA | oreberries                           | 0.5.0                    | oreberries-1.12.x-0.5.0.jar                    | None                                     |
       | LCHIJA | packcrashinfo                        | %VERSION%                | packcrashinfo-1.0.1.jar                        | None                                     |
       | LCHIJA | parry                                | 1.0                      | parry-1.0-hotfix.jar                           | None                                     |
       | LCHIJA | patchouli                            | 1.0-23.6                 | Patchouli-1.0-23.6.jar                         | None                                     |
       | LCHIJA | placeableitems                       | 3.3                      | placeableitems-3.3.jar                         | None                                     |
       | LCHIJA | traverse                             | 1.6.0                    | Traverse-1.12.2-1.6.0-69.jar                   | None                                     |
       | LCHIJA | pvj                                  | 1.6.4                    | ProjectVibrantJourneys-1.12.2-1.6.4.jar        | None                                     |
       | LCHIJA | rare-ice                             | ${version}               | rare-ice-0.1.0.jar                             | None                                     |
       | LCHIJA | reborncore                           | 3.19.5                   | RebornCore-1.12.2-3.19.5-universal.jar         | None                                     |
       | LCHIJA | reccomplex                           | 1.4.8.2                  | RecurrentComplex-1.4.8.2.jar                   | None                                     |
       | LCHIJA | xreliquary                           | 1.12.2-1.3.4.796         | Reliquary-1.12.2-1.3.4.796.jar                 | None                                     |
       | LCHIJA | roguelike                            | 2.4.0                    | RoguelikeDungeonsFnarEdition-1.12.2-2.4.0.jar  | None                                     |
       | LCHIJA | savemystronghold                     | 1.12.2-1.0.0             | savemystronghold-1.12.2-1.0.0.jar              | None                                     |
       | LCHIJA | schematica                           | 1.8.0.169                | Schematica-1.12.2-1.8.0.169-universal.jar      | None                                     |
       | LCHIJA | secretroomsmod                       | 5.6.4                    | secretroomsmod-1.12.2-5.6.4.jar                | None                                     |
       | LCHIJA | skewers                              | 1.1                      | Skewers-1.1.jar                                | None                                     |
       | LCHIJA | soulshardsrespawn                    | 1.12.2-1.1.1-13          | SoulShardsRespawn-1.12.2-1.1.1-13.jar          | None                                     |
       | LCHIJA | spartanshields                       | 1.5.4                    | SpartanShields-1.12.2-1.5.4.jar                | None                                     |
       | LCHIJA | specialmobs                          | 1.0.4_for_mc1.12.2       | specialmobs-1.0.4_for_mc1.12.2.jar             | None                                     |
       | LCHIJA | lemonlib                             | 1.3.0                    | lemonlib-1.12.2-1.3.0.jar                      | None                                     |
       | LCHIJA | statues                              | 1.0.1                    | statues-rechiseled-1.12.2-1.0.1.jar            | None                                     |
       | LCHIJA | stg                                  | 1.12.2-1.2.3             | stg-1.12.2-1.2.3.jar                           | None                                     |
       | LCHIJA | storagedrawers                       | 5.2.2                    | StorageDrawers-1.12.2-5.4.2.jar                | None                                     |
       | LCHIJA | streams                              | 0.4.9                    | Streams-1.12-0.4.9.jar                         | None                                     |
       | LCHIJA | tinkersaddons                        | 1.0.7                    | Tinkers' Addons-1.12.1-1.0.7.jar               | None                                     |
       | LCHIJA | tinkersjei                           | 1.2                      | tinkersjei-1.2.jar                             | None                                     |
       | LCHIJA | tinkertoolleveling                   | 1.12.2-1.1.0.DEV.b23e769 | TinkerToolLeveling-1.12.2-1.1.0.jar            | None                                     |
       | LCHIJA | tombmanygraves                       | 1.12-4.2.0               | TombManyGraves-1.12-4.2.0.jar                  | None                                     |
       | LCHIJA | tombmanygraves2api                   | 1.12.2-1.0.0             | tombmanygraves2api-1.12.2-1.0.0.jar            | None                                     |
       | LCHIJA | tombmanypluginsbackpacks             | 1.0.0                    | tombmanypluginsbackpacks-1.0.0.jar             | None                                     |
       | LCHIJA | tombmanypluginscosmeticarmorreworked | 1.0.0                    | tombmanypluginscosmeticarmorreworked-1.0.0.jar | None                                     |
       | LCHIJA | torchmaster                          | 1.8.4.84                 | torchmaster_1.12.2-1.8.4.84.jar                | 5e9a436b366831c8f54a7e80b015784da69278c6 |
       | LCHIJA | translocators                        | 2.5.2.81                 | Translocators-1.12.2-2.5.2.81-universal.jar    | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
       | LCHIJA | tumbleweed                           | 1.12-0.4.7               | tumbleweed-1.12-0.4.7.jar                      | None                                     |
       | LCHIJA | unloader                             | 1.2.0                    | unloader-1.2.0.jar                             | None                                     |
       | LCHIJA | vanillafix                           | 1.0.10-150               | VanillaFix-1.0.10-150.jar                      | None                                     |
       | LCHIJA | wanionlib                            | 1.12.2-2.5               | WanionLib-1.12.2-2.5.jar                       | None                                     |
       | LCHIJA | waystones                            | 4.1.0                    | Waystones_1.12.2-4.1.0.jar                     | None                                     |
       | LCHIJA | craftstudioapi                       | 1.0.0                    | CraftStudio-1.0.1.95-mc1.12-alpha.jar          | None                                     |
       | LCHIJA | cyclopstek                           | 1.0.34                   | cyclopstek-1.0.34.jar                          | None                                     |
       | LCHIJA | llibrary                             | 1.7.20                   | llibrary-1.7.20-1.12.2.jar                     | b9f30a813bee3b9dd5652c460310cfcd54f6b7ec |
       | LCHIJA | modwinder                            | 1.1                      | Ladylib-2.3.jar                                | None                                     |
       | LCHIJA | ladylib                              | 2.3                      | Ladylib-2.3.jar                                | None                                     |
       | LCHIJA | lumen                                | 0.4.1                    | lumen-Beta 0.4.2.jar                           | None                                     |
       | LCHIJA | gravelores                           | 1.8                      | gravelores-1.8.jar                             | None                                     |
       | LCHIJA | mysticallib                          | 1.12.2-1.13.0            | mysticallib-1.12.2-1.13.0.jar                  | None                                     |
       | LCHIJA | unidict                              | 1.12.2-3.0.8             | UniDict-1.12.2-3.0.8.jar                       | None                                     |
        Loaded coremods (and transformers): SplashAnimationCoremod (SplashAnimation-0.2.1.jar)
                                        pl.asie.splashanimation.core.SplashProgressTransformer
                                      RandomPatches (randompatches-1.12.2-1.22.1.10.jar)
                                        com.therandomlabs.randompatches.core.RPTransformer
                                      OpenEyePlugin (OpenEye-1.12.2-0.8.jar)
                                        openeye.asm.MultiTransformer
                                      ForgelinPlugin (Forgelin-1.8.4.jar)
                                        
                                      Do not report to Forge! (If you haven't disabled the FoamFix coremod, try disabling it in the config! Note that this bit of text will still appear.) (foamfix-0.10.14-1.12.2.jar)
                                        pl.asie.foamfix.coremod.FoamFixTransformer
                                      DupeFixProjectCoreMod (DupeFixProject-1.12.2-3.1.6.jar)
                                        
                                      OpenModsCorePlugin (OpenModsLib-1.12.2-0.12.2.jar)
                                        openmods.core.OpenModsClassTransformer
                                      PhosphorFMLLoadingPlugin (phosphor-forge-mc1.12.2-0.2.7-universal.jar)
                                        
                                      Quark Plugin (Quark-r1.6-179.jar)
                                        vazkii.quark.base.asm.ClassTransformer
                                      ObfuscatePlugin (obfuscate-0.4.2-1.12.2.jar)
                                        com.mrcrayfish.obfuscate.asm.ObfuscateTransformer
                                      FarseekCoreMod (Farseek-1.12-2.5.1.jar)
                                        farseek.core.FarseekClassTransformer
                                      ColytraLoadingPlugin (colytra-1.12.2-1.2.0.4.jar)
                                        c4.colytra.asm.ElytraTransformer
                                      Born in a Barn (Born In A Barn V1.8-1.12-1.1.jar)
                                        com.chocohead.biab.BornInABarn
                                      llibrary (llibrary-core-1.0.11-1.12.2.jar)
                                        net.ilexiconn.llibrary.server.core.plugin.LLibraryTransformer
                                        net.ilexiconn.llibrary.server.core.patcher.LLibraryRuntimePatcher
                                      VanillaFixLoadingPlugin (VanillaFix-1.0.10-150.jar)
                                        
                                      ChunkGenLimiterCoremod (chunkgenlimiter-1.1-core.jar)
                                        io.github.barteks2x.chunkgenlimiter.coremod.ChunkGenLimitTransformer
                                      SecretRoomsMod-Core (secretroomsmod-1.12.2-5.6.4.jar)
                                        com.wynprice.secretroomsmod.core.SecretRoomsTransformer
                                      IvToolkit (IvToolkit-1.3.3-1.12.jar)
  OpenModsLib class transformers: [llama_null_fix:FINISHED],[horse_base_null_fix:FINISHED],[pre_world_render_hook:ENABLED],[player_render_hook:ENABLED],[horse_null_fix:FINISHED]
  Pulsar/tconstruct loaded Pulses: - TinkerCommons (Enabled/Forced)
                                   - TinkerWorld (Enabled/Not Forced)
                                   - TinkerTools (Enabled/Not Forced)
                                   - TinkerHarvestTools (Enabled/Forced)
                                   - TinkerMeleeWeapons (Enabled/Forced)
                                   - TinkerRangedWeapons (Enabled/Forced)
                                   - TinkerModifiers (Enabled/Forced)
                                   - TinkerSmeltery (Enabled/Not Forced)
                                   - TinkerGadgets (Enabled/Not Forced)
                                   - TinkerOredict (Enabled/Forced)
                                   - TinkerIntegration (Enabled/Forced)
                                   - TinkerFluids (Enabled/Forced)
                                   - TinkerMaterials (Enabled/Forced)
                                   - TinkerModelRegister (Enabled/Forced)
                                   - chiselIntegration (Enabled/Not Forced)
                                   - chiselsandbitsIntegration (Enabled/Not Forced)
                                   - wailaIntegration (Enabled/Not Forced)
                                   - quarkIntegration (Enabled/Not Forced)
  Pulsar/natura loaded Pulses: - NaturaCommons (Enabled/Forced)
                               - NaturaOverworld (Enabled/Not Forced)
                               - NaturaNether (Enabled/Not Forced)
                               - NaturaDecorative (Enabled/Not Forced)
                               - NaturaTools (Enabled/Not Forced)
                               - NaturaEntities (Enabled/Not Forced)
                               - NaturaOredict (Enabled/Forced)
                               - NaturaWorld (Enabled/Not Forced)
  Modpack Information: Modpack: [Hexxit II] Version: [1.0.0-Server] by author [Jon]
  RebornCore: Plugin Engine: 0
              RebornCore Version: 3.19.5
              Runtime Debofucsation 1
              Invalid fingerprint detected for RebornCore!
  Suspected Mods: Rare Ice (rare-ice)
```

</details>

**Describe the issue**

This error keeps occurring as I travel through the world. Not sure what is causing it or how to fix it.

**Comment by xJon** [comment]
Full game log from #34 over https://github.com/xJon/Hexxit-II/issues/34#issuecomment-955740963.

**Comment by xJon** [comment]
This should be fixed in `1.1.0`!
